package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@modusoperandi/eslint-config",
+  "version": "1.0.0",
+  "description": "Module containing the lint configuring for MO",
+  "main": "index.js",
+  "author": "Tyler Newton (tnewtonMo)",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MO-Movia/eslint-config.git"
+  },
+  "dependencies": {
```

Comment:
I'm pretty sure all of these need to be peer.

Reply:
As per eslint documentation, all plugins should be dependencies.